image-files/config/ibc/config.ini.tmpl

```diff
@@ -788,7 +788,7 @@ DismissNSEComplianceNotice=yes
 # the command server
 
 #CommandServerPort=7462
-CommandServerPort=0
+CommandServerPort=${COMMANDSERVERPORT}
```

Comment:
sorry, this file is OK to go, except for this line. as this is related to TWS. please take it out for the time being. as this needs to go together witht the telnet piece